en/docs/quick-starts/age-based-adaptive-auth.md

```diff
@@ -22,28 +16,27 @@ The instructions below guide you through specifying authentication steps based o
 
 ## Configure claims
 
-1.  Start the server and log in to the [management console](insertlink).
+1.  Start the server and log in to the Management Console (`https://<IS_HOST>:<PORT>/carbon`).
 
-2.  Click **List** under **Claims** and click
-    [http://wso2.org/claims](https://localhost:9443/carbon/identity-claim-mgt/list-local-claims.jsp).
+2.  Click **List** under **Claims** and click http://wso2.org/claims (`https://localhost:9443/carbon/identity-claim-mgt/list-local-claims.jsp`).
```

Comment:
`https://localhost:9443/carbon/identity-claim-mgt/list-local-claims.jsp` IMO we shouldn't mention this.